src/main/java/plugily/projects/villagedefense/arena/managers/ScoreboardManager.java

```diff
@@ -61,7 +67,11 @@ public ScoreboardManager(Arena arena) {
    * @see User
    */
   public void createScoreboard(User user) {
-    Scoreboard scoreboard = ScoreboardLib.createScoreboard(user.getPlayer()).setHandler(new ScoreboardHandler() {
+    Player player = user.getPlayer();
+    lastBoardMap.put(player.getUniqueId(), player.getScoreboard());
+    player.setScoreboard(dummyScoreboard);
```

Comment:
Why do we need to set a new scoreboard for players?

Reply:
Ask whoever do the original code. I only copy it here.

Reply:
Revert now before breaking the scoreboard behaviour.

Reply:
Fine. I'll remove it. Why should I explain on your code?

Reply:
I am sure it was there to eliminate conflicts with other scoreboards

Reply:
It does not affect much. We can leave it there and see if any user has the problem then we will add back.

Reply:
wait, what, I'm totally idiot, I realized that I was who added this to set the player board to a new scoreboard, so [this](https://github.com/Plugily-Projects/Village_Defense/pull/119/files#diff-12443f56c0550a13763279516561ea27606c13521ec8ee7ba4c4332649f1510dL120-L122) implementation is good

Reply:
Just to be safe, I've rolled back the removal, since the lastBoard will always non-null as the player always has the default scoreboard called Main Scoreboard.